arq/worker.py

```diff
@@ -153,6 +154,9 @@ class Worker:
     :param after_job_end: coroutine function to run after job has ended and results have been recorded
     :param handle_signals: default true, register signal handlers,
       set to false when running inside other async framework
+    :param wait_for_job_completion_on_signal_seconds: time to wait before cancelling tasks after a signal.
+      Useful together with `terminationGracePeriodSeconds` in kubernetes,
+      when you want to make the pod complete jobs before shutting down
```

Comment:
Should probably mention here that it also won't pick new tasks

Reply:
maybe, but this is good enough for now.